Some More Fixes

Fiery36 opened the pull request:
I'm running out of name ideas for these.

Fiery36 committed:
removed home block oredict qb task because of JEI issue

Fiery36 committed:
removed odd reward

Fiery36 committed:
fixed techguns shield repair mats

Fiery36 committed:
correction

Fiery36 committed:
spelling

Fiery36 committed:
impossible

Fiery36 committed:
ebonite now requires vulcanizing press quest instead of rubber quest

Fiery36 committed:
fixed broken carbon black quest

Fiery36 committed:
removed lapis lazuli quest as a requirement for crowbar

you don't need lapis to make it

planetme commented on config/betterquesting/resources/supersymmetry/lang/en_us.lang:
```diff
@@ -1971,7 +1971,7 @@ susy.quest.db.796249166.desc=- §6Lever Module§r: Works as a simple lever, both
 susy.quest.db.797571327.desc=The §cShell Higher Olefin Process§r is a method to produce linear alpha olefins. The catalyst is the main hassle in the process, and requires multitudinous steps to synthesize. Good luck, you will need it.%n%nThese linear alpha olefins are essential for producing extraction agents for ore processing in the EV age, and are also useful for producing lubricant base oils  ...
 susy.quest.db.802655963.title=Combustion Generators
-susy.quest.db.802655963.desc=Unlocked in EV, these combustion generators are a compact way to produce power from refined fuels without the need for and oxidant and a large multiblock! These are rather expensive and require 2 iridium each.
+susy.quest.db.802655963.desc=Unlocked in EV, these combustion generators are a compact way to produce power from refined fuels without the need for an oxidant and a large multiblock! These are rather expensive and require 2 iridium each.
```
this is actually inaccurate rn, the LV and MV combustion generators are unlocked in MV using 2 palladium and the HV in EV using 2 iridium

Fiery36 committed:
nylon quest -> Ni-Zn battery quest instead of MV battery hull quest

Fiery36 committed:
war never changes

Fiery36 committed:
removed bolt action crafting table recipe

the only way you could craft it pre-LV was if you looted bullets from structures anyway

Fiery36 committed:
updated carbon quest

Fiery36 committed:
specify that extruders make parts more efficiently

Fiery36 committed:
shorten

Fiery36 committed:
electrolytic cell no longer requires LV electrolyzer quest

Fiery36 committed:
inform

Fiery36 committed:
decrease item requirement for sintering oven brick to 1 to match

frankly this should be done for all multiblock quests

Fiery36 committed:
specify

Fiery36 committed:
removed mentions of residual deposits

Fiery36 committed:
tip

Fiery36 committed:
potassium chloride -> rock salt

Fiery36 committed:
more correcter

Fiery36 committed:
mention aluminium hydroxide seed crystal recipe

trainvoi commented on groovy/postInit/mod/TechGuns.groovy:
```diff
@@ -367,12 +368,6 @@ crafting.replaceShaped("techguns:revolver", item('techguns:revolver'), [
     [ore('craftingToolFile'), item('techguns:itemshared', 42), item('techguns:itemshared', 1)]
 ])
 
-crafting.replaceShaped("techguns:boltaction", item('techguns:boltaction'), [
```
why is this recipe deleted.

Fiery36 replied:
It seems kind of weird to have it be possible to craft the gun before the bullets. The only way you can craft it pre-LV is if you loot bullets from structures.

trainvoi replied:
bandit mobs do drop bullets in their death, and bandit shack has tons of bullets too, so it's worth it.

Fiery36 replied:
It's definitely worth it because the bolt action is way better than other pre-LV guns, but that's not my point. I'm saying that logic wise being able to craft a gun before it's bulets makes no sense.

Fiery36 committed:
remove silver oxide quest and moved info to silver quest

Fiery36 committed:
add mention of stearic acid as the fourth LV option

bruberu committed:
fix: review

bruberu committed:
fix: remove evil recipe
